Tests/ImagineEngineTests/CameraTests.swift

```diff
@@ -74,6 +74,20 @@ final class CameraTests: XCTestCase {
         scene.camera.position = Point(x: -2000, y: -1000)
         XCTAssertEqual(scene.camera.position, Point(x: -2000, y: -1000))
     }
+    
+    func testResizeEventTrigger() {
+        let scene = Scene(size: Size(width: 500, height: 300))
+        var resizeTriggerCount = 0
+        scene.camera.events.resized.observe { resizeTriggerCount += 1 }
+        
+        XCTAssertEqual(scene.camera.rect, Rect(x: 250, y: 150, width: 0, height: 0))
+        XCTAssertEqual(resizeTriggerCount, 0)
+        
+       // set view frame and add scene to game thus resizing the camera
+        game.view.frame.size = Size(width: 250, height: 150)
+        game.scene = scene
+        XCTAssertEqual(scene.camera.rect, Rect(x: 125, y: 75, width: 250, height: 150))
+        XCTAssertEqual(resizeTriggerCount, 1)
```

Comment:
How about setting the size to the same value again and making sure that we don't get an additional trigger?